src/main/scala/mesosphere/marathon/api/v2/validation/PodsValidation.scala

```diff
@@ -194,7 +195,7 @@ trait PodsValidation {
     hostPorts.distinct.size == hostPorts.size
   }
 
-  def podValidator(enabledFeatures: Set[String], mesosMasterVersion: SemanticVersion): Validator[Pod] = validator[Pod] { pod =>
+  def podValidator(enabledFeatures: Set[String], mesosMasterVersion: SemanticVersion, defaultNetworkName: ScallopOption[String]): Validator[Pod] = validator[Pod] { pod =>
```

Comment:
please don't use scallop here, it's not needed. use a simple `Option[String]` type instead. pretty sure that you can invoke `.get` on a `ScallopOption[T]` to get an `Option[T]` (to pass in here)